cnf-tests/.konflux/redhat.repo

```diff
@@ -0,0 +1,27 @@
+[rhel-9-for-$basearch-appstream-beta-rpms]
```

Comment:
For RHEL EUS releases like 9.4 EUS, the EUS RPM repositories should also be included no?
Otherwise the container may not pick up the latest CVE fixes.

Reply:
good question, but do we want to build the image with EUS base? how do we know that?

Reply:
There is no EUS base image. There is only latest 9.4 which is non-EUS to the best of my knowledge. That is why I think we have to find a way to include the EUS repositories (when applicable) and maybe even run a 'dnf update' so that we get from 9.4 to 9.4 EUS plus installing whichever RPMs we need from EUS repositories.

Reply:
I have added the EUS repos but for now it fails to install the packages for some reason that is why we don't see that in the lockfile. I believe this step is anyway advanced and should not block this PR considering we are speaking about extended update support for rhel. Also konflux bot will be periodically updating the deps automatically including the bases and soon lockfiles. We can file a card on the konclux side to support aligning our operators with the OCP deps using some param or something similar to indicate that.

Reply:
EUS repo support:
https://github.com/openshift-kni/cnf-features-deploy/pull/2285